castor/webhooks/signals.py

```diff
@@ -0,0 +1,14 @@
+from django.db.models.signals import post_save
+from django.dispatch import receiver
+
+from docker_events.models import DockerEvent
+from webhooks.tasks import dispatch_docker_event
+
+
+@receiver(post_save, sender=DockerEvent)
+def dispatch_new_docker_events(sender, **kwargs):
+    """
+    Every time a new Docker event is being saved in the database, dispatch it.
+    """
+    if kwargs['created']:
+        dispatch_docker_event.delay(kwargs['instance'].id)
```

Comment:
If events are dispatched only when a new event is created, how does this related to `--resume` in the worker? Shouldn't events between the resumed event and the latest event already existing in the database be dispatched too?

That's more like a "How do we imagine `--resume` to work" and might not be a bug.